### ISSUE TITLE
False error about incompatible override of private(double-underscore) variables

### ISSUE THREAD

**Issue by Azureblade3808** [microsoft/pylance-release#863]
## Environment data

-   Language Server version: Pylance 2021.1.2 / Python 2021.1.502429796
-   OS and version: Windows 7
-   Python version (& distribution if applicable, e.g. Anaconda): 3.8.5(Anaconda)

## Actual behavior

```python
class Base:
    __value: int = 0

    _ = __value   # To suppress "unused" warning.


class Derived(Base):
    __value: str = ""
    # Error:
    # "__value" overrides symbol of same name in class "Base"
    #   "str" is incompatible with "int"  Pylance(reportIncompatibleVariableOverride)

    _ = __value   # To suppress "unused" warning.
```

## Expected behavior

No error

## Reason

Same with #725 , private(double-underscore) variables should not interfere with each others.

**Comment by Azureblade3808** [comment]
I think this issue is probably related with the fix of #843 .

**Comment by jakebailey** [comment]
Yeah, this should be working per #725.

**Comment by erictraut** [comment]
This is not the same as #725, which was about the use of `Final`. This appears to still be an issue. I'll look into it.

**Comment by jakebailey** [comment]
Doh. Of course.

**Comment by Azureblade3808** [comment]
After reading the changed lines of #843 , I do agree that it is not related.

**Comment by erictraut** [comment]
This will be fixed in the next release of pylance.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.1.3, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202113-27-january-2021
